api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java

```diff
@@ -59,22 +59,14 @@
   private static DbUser dbUser;
 
   void applyTimestampUpdateToUser(
-      UserDao userDao,
       AccessModuleService accessModuleService,
       String username,
       AccessModuleName moduleName,
       @Nullable Timestamp timestamp,
       boolean isBypass) {
     accessModuleService.updateCompletionTime(dbUser, moduleName, timestamp);
 
-    if (moduleName == AccessModuleName.PROFILE_CONFIRMATION) {
```

Comment:
the special-case here is "don't try to bypass `PROFILE_CONFIRMATION` because it is not bypassable